File: C#/Easy/IsAnagram.cs
public class Solution
{
    public bool IsAnagram(string s, string t)
    {
        char[] sCharArr = s.ToCharArray();
        char[] tCharArr = t.ToCharArray();

        Array.Sort(sCharArr);
        Array.Sort(tCharArr);

        return Enumerable.SequenceEqual(sCharArr, tCharArr);
    }
}

File: Java/Easy/IsAnagram.java
import java.util.Arrays;

public class Main
{
    public boolean isAnagram(String s, String t)
    {
        char[] sCharArray = s.toCharArray();
        char[] tCharArray = t.toCharArray();

        Arrays.sort(sCharArray);
        Arrays.sort(tCharArray);

        return Arrays.equals(sCharArray, tCharArray);
    }
}